### DISCUSSION TITLE
【求助】9.20 充值，现在还没有升级到专业版，帮我看一下

### DISCUSSION THREAD

**Discussion by xelthy** [getlantern/lantern-forum-cn#883] [General]
9.20 充值，现在还没有升级到专业版，帮我解决一下
![WechatIMG149](https://github.com/user-attachments/assets/ecbb3ff4-084d-47d4-9f02-672829bab91e)

**Comment by MaxMKMax** [comment]
@xelthy 好的. 请您也提供我账号邮箱.
已经帮您提交问题上去了. 请您放心, 会尽快给您解决问题.

**Reply by xelthy** [reply]
xelthy@163.com 
969914769@qq.com
好像是这两个中的一个，谢谢^_^

**Reply by greySkiesMarsh** [reply]
Hey @xelthy 

I saw accounts in our system related to both of those email addresses, so I went ahead and upgraded them both just in case.

**Reply by xelthy** [reply]
Thanks! OK now

**Reply by MaxMKMax** [reply]
@xelthy You are welcome.
